en/collect/import-using-online-bibliographic-database/slr.md

```diff
@@ -0,0 +1,59 @@
+# Conducting automated cross-library search
+
+## Providing a study file for automated search
+
+To provide automated search JabRef needs the set of targeted digital libraries and the used queries.
+To provide these a `study.bib` file, also called study definition, has to be created.
```

Comment:
How this file is created?
Do the specific entry types (@Study, @SearchQuery, @Library) have to be created into JabRef by the user through "Options -> Customize entry types" ?

Reply:
Hey there!
In the current version, these entries are created manually :).
But as we will switch to YAML this will be irrelevant.